### ISSUE TITLE
Issue importing CSV file

### ISSUE THREAD

**Issue by aweaver1fandm** [UniTime/unitime#89]
[unitime.txt](https://github.com/UniTime/unitime/files/6804090/unitime.txt)
I have a user trying to import some data as a CSV file (I renamed the file here to be able to upload, original file name was:
Unitime Class File- test.csv

When they do that they get a null pointer exception (See image that hopefully uploaded)
![image](https://user-images.githubusercontent.com/55799947/125348925-da910200-e32a-11eb-8a18-d77070740040.png)
The unitime logfile on the server has almost no other information that is different from what is on the screen...

[07/12/21 15:23:23] INFO   JProf -> Using wall time.
[07/12/21 15:23:39] INFO   TilesRequestProcessor -> Tiles definition factory found for request processor ''.
[07/12/21 15:23:45] INFO   Registration -> Registration information received.
[07/12/21 15:24:23] INFO   QueueProcessor -> Queue processor is up and running.
[07/12/21 15:24:23] INFO   QueueProcessor -> Waiting for tasks to run...
[07/12/21 15:24:40] INFO   QueueProcessor -> Executing Course Offerings: Generate XML
[07/12/21 15:24:40] ERROR  ScriptExecution -> Execution failed: 
java.lang.NullPointerException
	at org.unitime.timetable.server.script.ScriptExecution.execute(ScriptExecution.java:151)
	at org.unitime.timetable.util.queue.QueueItem.executeItem(QueueItem.java:104)
	at org.unitime.timetable.server.script.ScriptExecution.executeItem(ScriptExecution.java:308)
	at org.unitime.timetable.util.queue.LocalQueueProcessor$RunningItem.run(LocalQueueProcessor.java:247)
[07/12/21 15:24:40] INFO   QueueProcessor -> Task Course Offerings: Generate XML failed (null).

Any ideas what may be causing this issue?

**Comment by says0528** [comment]
It looks to me like jython library may be missing from the tomcat lib directory.  Can you check to see if the file "jython-standalone-2.5.3.jar” exists in the lib folder for the tomcat you are using?  Let me know if it is missing and I can help you obtain a copy.

Thanks,
Stephanie

**Comment by aweaver1fandm** [comment]
I copied the jar file into /var/lib/tomcat9/lib, chowned it appropriately and that seems to have corrected that issue but then the script processing dies with the following.  If I had to guess it looks like it may be a data issue where the code expects a number but isn't getting one so I'm looking more into it
![image](https://user-images.githubusercontent.com/55799947/125468768-03bd5117-65a5-4b6a-8748-52824bc414bf.png)

**Comment by says0528** [comment]
Based on the error, the problem is likely a badly formatted start time string for a section.

**Comment by aweaver1fandm** [comment]
Closing the original issue as resolved by having the jython file.  I will say that the few places I saw build/install instructions, none of them mentioned the need for the Jython file.  The latest error I am working through and may have resolved.  One of the classes had start time set to 0 and end time set to 1.  If fixing those does not resolve the issue I will keep looking at the data.  Thank you for the help
